runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -193,6 +212,10 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 		{
 			ladders.add(gameObject);
 		}
+		if (BARROWS_SARCOPHAGI.contains(gameObject.getId()))
+		{
+			isSarcophagusPresent = true;
```

Comment:
This seems needlessly complex, since all of the sarcophagi are in one region, you can just check the current region

Reply:
What do you mean by region? Like checking the plane of a player?

Reply:
If you get the worldloc of the player and getRegionID() there is one specific to barrows. It is a 64x64 map square.